src/contentMetaData.js

```diff
@@ -0,0 +1,1154 @@
+                {
+                    name: 'Instructors',
+                    short_name: 'INSTRUCTORS',
+                    is_group_by: true,
+                    value: 'instructor',
+                },
+                {
+                    name: 'Genres',
+                    short_name: 'Genres',
+                    is_group_by: true,
+                    value: 'genre',
+                },
+            ],
+        },
+        'rudiment': {
+            name: 'Rudiments',
+            icon: 'icon-drums',
+            description: "The 40 drum rudiments are essential for any drummer, no matter the style, genre, or scenario. You can use the videos below to help you learn, practice, and perfect every single one.",
+            allowableFilters: ['difficulty', 'genre', 'gear', 'topic'],
+            tabs: [
+                {
+                    name: 'All',
+                    short_name: 'ALL',
+                    value: '',
+                },
+                {
+                    name: 'Drags',
+                    short_name: 'DRAGS',
+                    is_required_field: true,
+                    value: 'topic[]->name match "drags"',
+                },
+                {
+                    name: 'Flams',
+                    short_name: 'FLAMS',
+                    is_required_field: true,
+                    value: 'topic[]->name match "flams"',
+                },
+                {
+                    name: 'Paradiddles',
+                    short_name: 'PARADIDDLES',
+                    is_required_field: true,
+                    value: 'topic[]->name match "paradiddles"',
+                },
+                {
+                    name: 'Rolls',
+                    short_name: 'ROLLS',
+                    is_required_field: true,
+                    value: 'topic[]->name match "rolls"',
+                },
+            ],
+            sortBy: 'sort',
+        },
+        'gear-guide': {
+            thumbnailUrl: 'https://dpwjbsxqtam5n.cloudfront.net/shows/gear.jpg',
+            name: 'Gear Guides',
+            icon: 'icon-shows',
+            description: "Drummers love their gear - and in here you will find videos on gear demos, reviews, maintenance, tuning tips and much more.",
+            allowableFilters: ['difficulty', 'genre'],
+            sortBy: '-published_on',
+            tabs: [
+                {
+                    name: 'Lessons',
+                    short_name: 'LESSONS',
+                    value: '',
+                },
+                {
+                    name: 'Instructors',
+                    short_name: 'INSTRUCTORS',
+                    is_group_by: true,
+                    value: 'instructor',
+                },
+                {
+                    name: 'Genres',
+                    short_name: 'Genres',
+                    is_group_by: true,
+                    value: 'genre',
+                },
+            ],
+        },
+        'challenges': {
```

Comment:
I believe this isn’t needed anymore since we’ve added the 'challenge' data type. Both would serve the same purpose.

Reply:
These are a distinct content type specifically for drumeo. Roxana would be a better person to ask regarding history. For the moment these will stay in the system, though I may remove them when we do the data import/update later depending on how things fit together.